bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx71.html

```diff
@@ -39,7 +39,7 @@ <h2 class="c-emphasis-box-title">
         </h2>
         <p>{{ _('The Firefox browser collects so little data about you, we don’t even require your email address. But when you use it to create a Firefox account, we can protect your privacy across more of your online life.') }}</p>
 
-        <p class="cta">{{ fxa_cta_link(entrypoint=_entrypoint, action='signup', button_text=_('Sign In'), button_class='mzp-c-button mzp-t-product mzp-t-small js-fxa-product-button', utm_campaign=_utm_campaign, utm_content='top-cta') }}</p>
+        <p class="cta">{{ fxa_cta_link(entrypoint=_entrypoint, action='signup', button_text=_('Sign In'), button_class='mzp-c-button mzp-t-product mzp-t-small js-fxa-product-button', utm_campaign=_utm_campaign, utm_content='top-cta', service_type=None) }}</p>
```

Comment:
We may want to apply this to the CRO version of this page also?

Reply:
oh whoops! will update now.